cloud-core/src/main/java/cloud/commandframework/execution/AsynchronousCommandExecutionCoordinator.java

```diff
@@ -76,10 +77,15 @@ private AsynchronousCommandExecutionCoordinator(
             final @NonNull CommandContext<C> commandContext,
             final @NonNull Queue<@NonNull String> input
     ) {
+        final CompletableFuture<CommandResult<C>> resultFuture = new CompletableFuture<>();
 
         final Consumer<Command<C>> commandConsumer = command -> {
             if (this.commandManager.postprocessContext(commandContext, command) == State.ACCEPTED) {
-                command.getCommandExecutionHandler().execute(commandContext);
+                try {
+                    command.getCommandExecutionHandler().execute(commandContext);
+                } catch (Throwable throwable) {
```

Comment:
No, no, no, no, no, no
Errors are not to be caught

Reply:
I don't see why not? Can you explain why players should see nothing show up when plugins have outdated dependencies, built against the wrong server version, or other such checked exceptions at runtime?

Reply:
`Error`s should not be caught in reasonable applications, as the javadoc of it says: https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html